recipes/rtd-poetry/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "rtd-poetry" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21f6d5bc8c2347b1d20c358fcba8f6052a637f3687a1a42393a42e75570ee67c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - intreehooks
```

Comment:
Does this actually need `intreehooks` at package build time?

Reply:
I guess the answer to this question is: "Yes, it does need `intreehooks` at build time." 😉